fe/fe-core/src/main/java/com/starrocks/transaction/PublishVersionDaemon.java

```diff
@@ -234,16 +234,26 @@ boolean publishTable(Database db, TransactionState txnState, TableCommitInfo tab
                 LOG.info("Removed partition {} from transaction {}", partitionId, txnId);
                 continue;
             }
-            if (!publishPartition(txnState, table, partition, partitionCommitInfo)) {
+            long currentTime = System.currentTimeMillis();
+            long versionTime = partitionCommitInfo.getVersionTime();
+            if (versionTime > 0) {
+                continue;
+            }
+            if (versionTime < 0 && currentTime + RETRY_INTERVAL_MS < Math.abs(versionTime)) {
```

Comment:
```suggestion
            if (versionTime < 0 && Math.abs(versionTime) + RETRY_INTERVAL_MS < currentTime) {
```

Reply:
Updated, Should be `versionTime < 0 && currentTime < Math.abs(versionTime) + RETRY_INTERVAL_MS)`